Exercise_B/index.html

```diff
@@ -3,14 +3,25 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>Exercise B</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="index.css" />
   </head>
   <body>
     <!-- CODE HERE -->
 
+   
+
+  <div id="blue" class="group"> </div>
+  <div id="green" class="group"> </div>
+
+
+
+
+      <div id="red"> </div>
+
+
     <script src="" async defer></script>
   </body>
 </html>
```

Comment:
Hello Chloe, 
Great work with this they are all positioned correctly. Good work with making use of CSS.